### PR TITLE
Added tests for dashboard api

### PR DESCRIPTION
**Pull request by intrepiditee**

### CODE REVIEW

File: import-automation/executor/test/test_dashboard_api.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
+import unittest
+from unittest import mock
+import subprocess
+
+from app.service import dashboard_api
+from app.service import iap
+
+
+@mock.patch('app.utils.pttime', lambda: '2020-07-15T12:07:17.365264-07:00')
+class DashboardAPITest(unittest.TestCase):
```
Comment (pradh) [inline import-automation/executor/test/test_dashboard_api.py:10]: Can this run in standalone() mode too?

File: import-automation/executor/test/utils.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    """
+    with open(expected_path, 'rb') as expected, open(path, 'rb') as file:
+        for i in range(num_lines):
+            line1 = expected.readline()
+            line2 = file.readline()
+            if line1 != line2:
+                print('WANT:', line1)
+                print('GOT:', line2)
+                return False
+    return True
```
Comment (pradh) [inline import-automation/executor/test/utils.py:45]: Add newline

File: import-automation/executor/test/test_dashboard_api.py (hunk 1 above)
Comment (intrepiditee) [reply import-automation/executor/test/test_dashboard_api.py:10]: Not tested. I would not expect it to

Comment (pradh) [reply import-automation/executor/test/test_dashboard_api.py:10]: expect it to ... what?

as a sanity check that it doesn't do API calls, I think its worth having.

Comment (intrepiditee) [reply import-automation/executor/test/test_dashboard_api.py:10]: Will add

File: import-automation/executor/test/utils.py (hunk 2 above)
Comment (intrepiditee) [reply import-automation/executor/test/utils.py:45]: added

File: import-automation/executor/test/test_dashboard_api.py (hunk 1 above)
Comment (intrepiditee) [reply import-automation/executor/test/test_dashboard_api.py:10]: added

**Pull request merged**